docs/source/run/parameters.rst

```diff
@@ -354,6 +354,15 @@ When both are specified, the per-species value is used.
     arrays of size ``sort_bin_size`` (+ guard cells) that are atomic-added to the main current
     arrays.
 
+* ``<plasma name> or plasmas.reorder_interval`` (`int`) optional (default `0`)
+    Reorder particles periodically to speed-up current deposition on GPU for a high-temperature plasma.
+    An interval of 1 means reordering is done on every slice.
+    To disable reordering set this to 0.
+
+* ``<plasma name> or plasmas.reorder_idx_type`` (2 `int`) optional (default `0 0` or `1 1`)
+    Change if plasma particles are binned to cells (0), nodes (1) or both (2)
+    for both x and y direction as part of the reordering.
```

Comment:
From the doc the default is not clear to me. Could you please add a sentence on that it is `hipace.depos_order_xy % 2, hipace.depos_order_xy % 2` and maybe also why so users understand how to choose it?